src/Behat/Gherkin/Lexer.php

```diff
@@ -325,7 +325,7 @@ protected function getKeywords($type)
                 foreach (explode('|', $keywords) as $keyword) {
                     $padded[] = false !== mb_strpos($keyword, '<', 0, 'utf8')
                         ? mb_substr($keyword, 0, -1, 'utf8') . '\s*'
-                        : $keyword . '\s+';
+                        : preg_quote($keyword) . '\s+';
```

Comment:
please don't duplicate the work of the existing PR, especially given that your fix is incomplete